.travis.yml

```diff
@@ -195,18 +195,21 @@ jobs:
         - $( npm bin )/wp-scripts test-e2e --config=./packages/e2e-tests/jest.config.js --listTests > ~/.jest-e2e-tests
         - $( npm bin )/wp-scripts test-e2e --config=./packages/e2e-tests/jest.config.js --cacheDirectory="$HOME/.jest-cache" --runTestsByPath $( awk 'NR % 4 == 3' < ~/.jest-e2e-tests )
 
+    - stage: deploy
+      name: Deploy Playground
+      env: INSTALL_WORDPRESS=false
+      before_deploy:
+        - npm install
+        - npm run playground:build -- --public-url '/gutenberg'
+        - npm run design-system:build
```

Comment:
It looks like with the changes applied in this PR, `design-system:build` will run twice on Travis:
```json
"playground:build": "npm run build:packages && parcel build playground/src/index.html -d playground/dist && npm run design-system:build",
```

Reply:
#17818